src/components/todos.js

```diff
@@ -0,0 +1,27 @@
+import { fetchAPI, METHOD } from "./constants";
+
+
+export const getTodolist = (userid)=> fetchAPI(`${userid}/items`); 
+
+export const addTodo = (userid, contents) => fetchAPI(`${userid}/items`, METHOD.POST, { contents }); 
```

Comment:
```suggestion
export const addTodoAPI = (userid, contents) => fetchAPI(`${userid}/items`, METHOD.POST, { contents }); 
```
스타일 또는 취향 차이일수도 있겠지만 내부 파일에서 addTodo를 보면 api함수인지 헷갈릴꺼 같습니다.